e2e/api_test.go

```diff
@@ -118,7 +125,11 @@ func TestE2E_StatusEndpoints(t *testing.T) {
 			require.NoError(t, err)
 			defer resp.Body.Close()
 
-			require.Equal(t, resp.StatusCode, http.StatusOK)
+			require.Equal(t, tc.statusCode, resp.StatusCode)
+
+			if tc.statusCode != http.StatusOK {
+				return
+			}
```

Comment:
Prior, only 200 was accepted and `require` made sense. Now that the status code is a value to test for, I would suggest changing it `require` to `assert`.

Reply:
I agree, thanks for pointing that out!